src/components/layouts/collection-page-layout.tsx

```diff
@@ -143,7 +140,12 @@ const CollectionPageLayout: React.FunctionComponent<CoursePageLayoutProps> = ({
   const [clickable, setIsClickable] = React.useState(true)
   const {viewer} = useViewer()
   const viewerId = viewer?.id
-  const {data: isCourseCompleted} = useIsCourseCompleted(viewerId, course.slug)
+  const {data: completedCourses} = useCompletedCourses(viewerId)
+  const isCourseCompleted =
+    !isEmpty(completedCourses) &&
```

Comment:
I'd consider using status property versus an empty check and make sure `loadUserCompletedCourses` returns an empty array by default